colabs/intro/Report_API_Quickstart.ipynb

```diff
@@ -1,1599 +1,1799 @@
-    "            runsets=[\n",
-    "                wr.Runset(entity='openrlbenchmark', project='cleanrl', query='bigfish', groupby=['env_id', 'exp_name'])\n",
-    "            ],\n",
-    "            custom_run_colors={\n",
-    "                ('Run set', 'bigfish', 'ppg_procgen'): \"#2980b9\",\n",
-    "                ('Run set', 'bigfish', 'ppo_procgen'): \"#e74c3c\",\n",
-    "            }\n",
-    "        ),\n",
-    "        wr.H2('Sit officia inventore non omnis deleniti.'),\n",
-    "        wr.PanelGrid(\n",
-    "            panels=[\n",
-    "                wr.LinePlot(x='global_step', y=['charts/episodic_return'], smoothing_factor=0.85, groupby_aggfunc='mean', groupby_rangefunc='minmax', layout={'x': 0, 'y': 0, 'w': 12, 'h': 8}),\n",
-    "                wr.MediaBrowser(media_keys=\"videos\", num_columns=4, layout={'w': 12, 'h': 8}),\n",
-    "            ],\n",
-    "            runsets=[\n",
-    "                wr.Runset(entity='openrlbenchmark', project='cleanrl', query='starpilot', groupby=['env_id', 'exp_name'])\n",
-    "            ],\n",
-    "            custom_run_colors={\n",
-    "                ('Run set', 'starpilot', 'ppg_procgen'): \"#2980b9\",\n",
-    "                ('Run set', 'starpilot', 'ppo_procgen'): \"#e74c3c\",\n",
-    "            }\n",
-    "        ),\n",
-    "    ]\n",
-    ")\n",
-    "report.save()"
-   ]
-  }
- ],
- "metadata": {
-  "kernelspec": {
-   "display_name": "Python 3.9.12 ('py39')",
-   "language": "python",
-   "name": "python3"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 0
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "31J25LOy8w8B"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/wandb/examples/blob/master/colabs/intro/Report_API_Quickstart.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+        "<!--- @wandbcode{python-report-api} -->"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EVdqKrIQ8w8C"
+      },
+      "source": [
+        "<img src=\"http://wandb.me/logo-im-png\" width=\"400\" alt=\"Weights & Biases\" />\n",
+        "<!--- @wandbcode{python-report-api} -->"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VJNPnEIL8w8C"
+      },
+      "source": [
+        "## What is the Report API?\n",
+        "- Programmatically create and modify reports in Python, including support for editing blocks, panels, and runsets.\n",
+        "- Create report templates to reuse and share with others\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "U1ClzRrR8w8C"
+      },
+      "source": [
+        "## Quick Links\n",
+        "- [🚀 Quickstart Guide](#quickstart) (~5min)\n",
+        "- [❓ FAQ](#faq)\n",
+        "- [📌 Complete Examples](#complete_examples)"
```

Comment:
The "complete_examples"  html ID doesn't work. Changing it from #complete_examples to #complete worked.